classes/shaman.lua

```diff
@@ -133,6 +183,58 @@ local function registerClass(self)
         self:RegisterAura("elemental_focus_1", 1, 16246, "echo_of_the_elements", "Left", 1, 255, 255, 255, true);
         self:RegisterAura("elemental_focus_2", 2, 16246, "echo_of_the_elements", "Left + Right (Flipped)", 1, 255, 255, 255, true);
     end
+    
+    if self.IsCata() then
+        self:RegisterAura("elemental_focus_1", 1, 16246, "genericarc_05", "Left", 1.5, 255, 255, 255, true);
+        self:RegisterAura("elemental_focus_2", 2, 16246, "genericarc_05", "Left + Right (Flipped)", 1.5, 255, 255, 255, true);

```

Comment:
After the PR, I will try something like this:
```lua
self:CreateEffect(
    "elemental_focus",
    SAO.CATA,
    16246, -- Clearcasting (buff)
    "aura",
    {
        overlays = {
            { stacks = 1, texture = "genericarc_05", position = "Left", scale = 1.5, option = false },
            { stacks = 2, texture = "genericarc_05", position = "Left + Right (Flipped)", scale = 1.5, option = { setupStacks = 0, testStacks = 2 } },
        },
    }
);
```
It looks more readable, isn't it?

It also removes the necessity to add overlay options and glow options manually later on.

(it requires recent devs, so you may not be able to try it before the PR is merged)